### ISSUE TITLE
Use libation to convert and add metadata to non audible audiobooks

### ISSUE THREAD

**Issue by hczv** [rmcrackan/Libation#726]
**Describe the solution you'd like**
Extend Libation functionality to also work with a local library.

Point Libation to a folder, let it scan and list all audiobooks in the folder and related metadata it found in them.
Then be able to convert the files in the folder and map an audiobook to an ASIN to populate it with metadata from audnex.us, create metadata.json and add cover to audiobook.

Alot like how it already works with audible, with limited manual intervention to map audiobooks.

**Describe alternatives you've considered**
Manually managing library

**Comment by manalive** [comment]
you can do most of this with mp3tag + audible source files right now. 
https://github.com/seanap/Plex-Audiobook-Guide/tree/master/mp3tag/data/sources

**Comment by hczv** [comment]
Yes, this is what im currently using to manage audiobook metadata, and it works.

I find Libation could be a very good fit for the same kind of thing

**Comment by rmcrackan** [comment]
This is one of a number of far-reaching features that have been given a lot of thought for a long time with no commitments. I think it's fair to say we're not intending to do it in the near term.
